### ISSUE TITLE
WGS84 to LAEA coordinate reprojection X is ~200m off

### ISSUE THREAD

**Issue by pergerch** [DotSpatial/DotSpatial#815]
When reprojecting a coordinate from WGS84 to ETRS89 LAEA, the x coordinate is about 200m off, compared to the reprojection in the proj.4 C library.

DotSpatial version: 1.9
but including fixes #814 and fixes from issues #387, #568

### Expected behaviour

Reprojecting the coordinate 16.4 E, 48.2 N (Vienna, Austria) from WGS 84 to ETRS89 LAEA should result in 4796297.43 / 2807999.15. Which, when reprojected back to WGS84, returns in both proj.4 C library as well as when using DotSpatial.Projection the original coordinate. 

### Actual behaviour

However, DotSpatial.Projection returns the LAEA coordinate 4796095.328482 / 2807999.153948 which is about 200m off in X.

### Code sample

```
ProjectionInfo pStart = KnownCoordinateSystems.Geographic.World.WGS1984;
ProjectionInfo pEnd = KnownCoordinateSystems.Projected.Europe.ETRS1989LAEA;

// Vienna, Austria
var lon = 16.4;
var lat = 48.2;

double[] xy = new double[] { lon, lat };
double[] z = new double[] { 0 };

Reproject.ReprojectPoints(xy, z, pStart, pEnd, 0, 1);

// Result:
// xy[0] ==> 4796095.328482
// xy[1] ==> 2807999.153948

// Expected result:
// xy[0] ==> 4796297.43
// xy[1] ==> 2807999.15
```

**Comment by mogikanin** [comment]
@pergerch please look at #730. Maybe here also missed girdshift file?

**Comment by pergerch** [comment]
Thank you for that quick reply. I doubt that this is the case. First of all the error only occurs for the X coordinate, the Y coordinate is perfectly fine. And the error only happens in the forward projections. If I instead use the X coordinate that Proj.4 in C outputs for the forward transformation, then the inverse (backward) transformation to WGS84 results in absolutely the same WGS84 coordinate that I used initially.

I already tried to analyze the code to find errors or discrepancies between the Proj.4 C library and DotSpatial.Projections, same as I did for #814 but unfortunately couldn't find any problem.
